corehq/ex-submodules/casexml/apps/stock/models.py

```diff
@@ -54,7 +65,7 @@ class StockTransaction(models.Model):
     # currently supported/expected: 'stockonhand', 'receipts', 'consumption'
     type = models.CharField(max_length=20)
     # e.g. 'loss', 'transfer', 'inferred'
-    subtype = TruncatingCharField(max_length=20, null=True, blank=True)
+    subtype = TruncatingTextField(max_length=40, null=True, blank=True)
```

Comment:
No need to truncate from a DB point of view, only necessary if we want to protect against someone dumping a huge string in there but that's also likely taken care of in other places.